Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Materials/InvisibleBacking.mat

```diff
@@ -0,0 +1,171 @@
+%YAML 1.1
```

Comment:
Do we need this even after turning on the `_ZWrite` values in the font assets below?

Reply:
The zwrite covers the fonts, not the unity UI which is covered by this